portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -119,20 +124,21 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     String message = request.getParameter(MESSAGE_PARAMETER);
     String nickname = request.getParameter(SENDER_PARAMETER);
     String postTag = request.getParameter(TAG_PARAMETER);
+    long parentID = PARENT_ID_PARAMETER;
```

Comment:
Any reason we need to define this "paraentID" local variable instead of directly using the PARENT_ID_PARAMETER variable?